src/index.js

```diff
@@ -3,16 +3,18 @@ import {
   Float32BufferAttribute
 } from 'three';
 
+import earcut from 'earcut';
+
+import interpolateLine from './interpolateLine';
+
+if(window !== 'undefined') {
 const THREE = window.THREE
   ? window.THREE // Prefer consumption from global THREE, if exists
   : {
   BufferGeometry,
   Float32BufferAttribute
 };
 
-import earcut from 'earcut';
```

Comment:
Please let's not change the order of the import statements, it's unrelated to this PR.